staging/istio/upgrade_operator.sh

```diff
@@ -57,7 +57,9 @@ cd "${TMPDIR}/${ISTIO_DASHBOARDS_PATH}" || exit
 
 for f in *; do
   rm -rf "${BASEDIR:?}"/${FORK_DASHBOARDS_PATH}"/${f}"
-  cp -R "$f" "${BASEDIR}/${FORK_DASHBOARDS_PATH}"
+  if [[ "$f" == *".json" && "$f" != "jsonnetfile"* ]]; then
```

Comment:
https://github.com/istio/istio/tree/1.23.2/manifests/addons/dashboards
I checked the upstream dashboard jsons and noticed they made a change to use jsonnet to generate some dashboards, so I took the newly generated ones into account but wanted to avoid the `.libsonnet` and jsonnet files since they are not dashboard jsons, so that's the new block I added here

Reply:
I also noticed that upstream had replaced the pilot-dashboard.json with a generated [pilot-dashboard.gen.json](https://github.com/istio/istio/blob/1.23.2/manifests/addons/dashboards/pilot-dashboard.gen.json) file so I removed our original one to avoid a dupe dashboard. Since this script does not do a full "replace" but rather is additive (maybe next time can look at doing a full copy/replace)